src/Modal/Modal.js

```diff
@@ -73,12 +75,14 @@ const ModalContent = createComponent({
   `,
 });
 
-function Modal({ children, title, animationDuration, showClose, onClose, open, ...props }) {
+function Modal({ children, title, animationDuration, showClose, onClose, open, blur, blurTarget, ...props }) {
```

Comment:
Let's default `blur = true` so that blur is opt-out rather than having to specify `blur` on every single modal in the web-app.

Reply:
👍